src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+
+  transform: scale(1);
+  transition: transform 300ms ease-in-out;
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-1;
+    margin: 0;
+    padding-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+    transition: transform 300ms ease-in-out;
+  }
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-left: 19px;
```

Comment:
The width and margin-left of the .card__photo element are using hardcoded pixel values. Check if these values can be replaced with relative units or variables for better scalability and maintainability of the styles.